src/Sylius/Bundle/CoreBundle/Processor/DummyCatalogPromotionProcessor.php

```diff
@@ -15,43 +15,35 @@
 
 use Doctrine\ORM\EntityManagerInterface;
 use Sylius\Bundle\CoreBundle\Applicator\CatalogPromotionApplicatorInterface;
+use Sylius\Component\Core\Model\CatalogPromotionInterface;
 use Sylius\Component\Core\Model\ProductInterface;
-use Sylius\Component\Core\Model\TaxonInterface;
-use Sylius\Component\Core\Repository\ProductRepositoryInterface;
-use Sylius\Component\Promotion\Model\CatalogPromotionInterface;
-use Sylius\Component\Taxonomy\Repository\TaxonRepositoryInterface;
+use Sylius\Component\Core\Provider\CatalogPromotionProductsProviderInterface;
 
 final class DummyCatalogPromotionProcessor implements CatalogPromotionProcessorInterface
```

Comment:
Should the name still be `Dummy`?

Reply:
Until we introduce some actions - it's still quite dummy with the hardcoded discount :smile: